src/libraries/Microsoft.Extensions.FileSystemGlobbing/ref/Microsoft.Extensions.FileSystemGlobbing.cs

```diff
@@ -247,14 +247,14 @@ public partial class PatternContextRaggedInclude : Microsoft.Extensions.FileSyst
         public override void Declare(System.Action<Microsoft.Extensions.FileSystemGlobbing.Internal.IPathSegment, bool> onDeclare) { }
         public override bool Test(Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoBase directory) { throw null; }
     }
-    public abstract partial class PatternContext<TFrame> : Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext
+    public abstract partial class PatternContext<TFrame> : Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext where TFrame : struct
     {
-        protected TFrame? Frame;
+        protected TFrame Frame;
```

Comment:
Couldn't we just leave this as `protected TFrame? Frame;` and just change `protected void PushDataFrame(TFrame? frame)` => `protected void PushDataFrame(TFrame frame)`?